src/camera_ros_example.cpp

```diff
@@ -97,7 +97,9 @@ int main(int argc, char** argv)
 
     const auto pose = lookat(camera_pos, look_at, Eigen::Vector3d(0,0,1));
 
-    const auto depth_img = sim.render(pose);
+    auto depth_img = sim.render(pose);
+
+    depth_img.data  = gl_depth_sim::noise(depth_img.data);
```

Comment:
Please add a ROS parameter that _optionally_ applies the noise. This should be something that people opt into.